src/main/java/com/amihaiemil/camel/StrictYamlSequence.java

```diff
@@ -0,0 +1,92 @@
+package com.amihaiemil.camel;
+
+import java.util.Collection;
+
+/**
+ * Decorator for a {@link YamlSequence} which throws YamlNodeNotFoundException
+ * if any of the methods of the decorated YamlSequence returns null
+ * (if the given index points to a YamlNode that is not a YamlMapping, for instance).
+ * @author Salavat.Yalalov
```

Comment:
@salikjan Here, add your fullname and email inside parathesis, and add ``@version $Id$`` bellow that.